<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="suresh-pc" name="Default test" tests="1" failures="1" timestamp="29 Jun 2016 10:58:26 GMT" time="2.058" errors="0">
  <testcase name="testcase1" time="2.039" classname="Topic9_Timeout_Testng.Example1_Timeout">
    <failure type="org.testng.internal.thread.ThreadTimeoutException" message="Method org.testng.internal.TestNGMethod.testcase1() didn&amp;apos;t finish within the time-out 2000">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method org.testng.internal.TestNGMethod.testcase1() didn't finish within the time-out 2000
]]>
    </failure>
  </testcase> <!-- testcase1 -->
</testsuite> <!-- Default test -->
